index.html

```diff
@@ -3,14 +3,20 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>TV Show Project | Hedyeh (Hedyeh65)</title>
```

Comment:
That`s an amazing work Hediyeh, congratulations. your js codes are very clear and all features work perfectly. you just need to add  "the data has (originally) come from TVMaze.com" anywhere on page. by the way, I hadnt done that as well:). and you may work on styling a bit more. adding an header text, logo etc. and responsiveness...